packages/map/src/components/DataTable.js

```diff
@@ -176,9 +176,9 @@ const DataTable = (props) => {
     const newTableColumns = [];
 
     Object.keys(columns).forEach((column) => {
-      if (columns[column].dataTable === true && '' !== columns[column].name) {
+      if (columns[column].dataTable === true && columns[column].name) {
         const newCol = {
-          Header: columns[column].label || columns[column].name,
+          Header: columns[column].label ? columns[column].label : columns[column].name,
```

Comment:
@Atash3000 I think this is the same thing, it looks like you swapped the short circuit evaluation for the ternary.